apps/confidential/confidential.go

```diff
@@ -400,40 +401,101 @@ func (cca Client) AuthCodeURL(ctx context.Context, clientID, redirectURI string,
 	return cca.base.AuthCodeURL(ctx, clientID, redirectURI, scopes, cca.base.AuthParams)
 }
 
+// WithTenantID specifies a tenant for a single authentication. It may be different than the tenant set in [New] by [WithAuthority].
```

Comment:
You might want to use the 1.19 doc comments to list which calls this works with.